banzai/data.py

```diff
@@ -123,12 +123,14 @@ def __imul__(self, value):
         return self
 
     def __itruediv__(self, value):
-        if isinstance(value, CCDData):
```

Comment:
I am not sure whether this should be CCDData or type(self), it seems to make sense to be type(self) instead of being restricted to the parent class.

Reply:
I disagree actually. Basically if the data object has an attribute uncertatinty we want to propagate it even if one is an image ccd data class  (e.g. from a bias frame) and the other is a spectrum ccddata object.

Reply:
Great, thanks for clarifying. I agree now.